lib/pub_sub/message.rb

```diff
@@ -11,20 +11,22 @@ def initialize(payload)
     end
 
     def process
-      validate_message!
+      begin
+        validate_message!
+      rescue PubSub::ServiceUnknown => e
+        return
+      end
       handler.process(data)
     end
 
     def validate_message!
       messages = PubSub.config.subscriptions[sender]
-      if messages.nil?
-        warning = "WARN: We received a message from #{sender} but we do " \
-                'not subscribe to that service.'
-        PubSub.logger.warn(warning)
-      end
-      unless messages.include?(type)
-        error = "We received a message from #{sender} but it was " \
-                "of unknown type #{type}."
+      if messages.blank?
+        warning = "#{PubSub.config.service_name} received a message from #{sender} but no matching subscription exists for that sender"
```

Comment:
We're not talking about outdated 80 char terminals from the 90s, this is my current setup, and it went from something that looked good, to something that looks bad.

It doesn't matter that it's only a simple error message, the automated wrapping makes it difficult to read.  Also, the error and log messages are the most common culprits for needing to be line wrapped, so this is exactly the case that needs to have a coding style standard applied to it.

The original author purposely split this message onto two lines precisely because he didn't want it auto wrapping, and many of our other projects go to great difficulty to ensure that long lines are split into multiple lines (see [here](https://github.com/westfield/consumption_service/blob/master/app/helpers/redis_client.js) for an example where every long line has been split onto multiple lines)

We used to have a Westfield Coding Guidelines document, which specified that all lines should be wrapped at 100 chars, but unfortunately this document was removed.